iaso/api/org_unit_change_requests/serializers.py

```diff
@@ -197,7 +197,6 @@ class OrgUnitChangeRequestRetrieveSerializer(serializers.ModelSerializer):
     class Meta:
         model = OrgUnitChangeRequest
         fields = [
-            "id",
```

Comment:
Nope, this one should stay. The one at line 272 should be remvoed.

Reply:
I am sorry I remove it by mistake instead the other one!